src/lib/layers/editable-geojson-layer.js

```diff
@@ -383,6 +407,22 @@ export default class EditableGeoJsonLayer extends EditableLayer {
     });
   }
 
+  handleFinishMovePosition(featureIndex: number, positionIndexes: number, groundCoords: number[]) {
+    const updatedData = this.state.editableFeatureCollection
+      .replacePosition(featureIndex, positionIndexes, groundCoords)
+      .getObject();
+
+    this.props.onEdit({
+      updatedData,
+      updatedMode: this.props.mode,
+      updatedSelectedFeatureIndex: featureIndex,
+      editType: 'finishMovePosition',
```

Comment:
Please move all these in an Object that is exported.

Reply:
All of the `handle` functions?

Reply:
@supersonicclay @matthrice Yes please do it for all of them.